sphinx/application/data/concepts/repositories/concept-repository-dashboard/src/main/java/chat/sphinx/concept_repository_dashboard/RepositoryDashboard.kt

```diff
@@ -48,6 +48,12 @@ interface RepositoryDashboard {
         challenge: String
     ): Response<Boolean, ResponseError>
 
+    suspend fun saveProfile(
```

Comment:
Rename to savePeopleProfile to be more descriptive

Reply:
resolved in a74452a